e2e/e2e-helper.js

```diff
@@ -64,6 +64,24 @@ export default class Helper {
     return fs.readJSONSync(bitJsonPath) || {};
   }
 
+  createPackageJson(
+    name: string = 'test',
+    version: string = '0.0.1',
+    packageJsonPath: string = path.join(this.localScopePath, 'package.json')
+  ) {
+    const packageJson = { name, version };
+    fs.writeJSONSync(packageJsonPath, packageJson);
+  }
+  changePackageManagerToYarn(
```

Comment:
this is confusing, it's change to yarn and also set the manage workspaces.
should be a seperate function.
Also we should add one for the useWorkspaces key.